package.json

```diff
@@ -61,7 +61,7 @@
     "test:all": "ember try:each"
   },
   "dependencies": {
-    "ember-app-scheduler": "^0.2.0",
+    "ember-app-scheduler": "kyleshay/ember-app-scheduler#64a0c91d8866b356439bb2d3068029ac59490090",
```

Comment:
Should we try `0.2.2` here?  Fyi - ember-app-scheduler is not the default anymore.

Reply:
Not sure what you mean "not the default anymore"
Was this dep removed?

Reply:
Still there.  Just you need to opt into ember-app-scheduler (see README).  There was a noticeable jump in the dummy app b/c [ember-app-scheduler](https://github.com/ember-app-scheduler/ember-app-scheduler#ember-app-scheduler) moves work until after first meaningful paint.  To get rid of that jump, we schedule scrollTop work in the `render` queue.  

Lmk if you notice or have any thoughts on that.  I could have missed something...

https://github.com/dollarshaveclub/ember-router-scroll/pull/101